infra/storage/file_transfer_managers/s3Mgr.class.php

```diff
@@ -122,9 +131,12 @@ protected function doLogin($sftp_user, $sftp_pass)
 			return false;
 		}
 
-		if($this->s3Arn && (!isset($sftp_user) || !$sftp_user) && (!isset($sftp_pass) || !$sftp_pass))
+		if($this->s3Arn && $this->s3Region && (!isset($sftp_user) || !$sftp_user) && (!isset($sftp_pass) || !$sftp_pass))
```

Comment:
Why did region become mandatory?   it can be null when calling getCacheCredentialsProvider